VAMobile/src/screens/HomeScreen/ProfileScreen/MilitaryInformationScreen/IncorrectServiceInfo/IncorrectServiceInfo.tsx

```diff
@@ -39,13 +39,7 @@ const IncorrectServiceInfo: FC<IncorrectServiceInfoScreenProps> = () => {
         <TextView accessibilityLabel={t('militaryInformation.incorrectServiceInfo.bodyA11yLabel.3')} variant="MobileBody" paragraphSpacing={true}>
           {t('militaryInformation.incorrectServiceInfo.body.3')}
         </TextView>
-        <ClickForActionLink
-          testID="incorrectServiceDMDCNumberTestID"
-          displayedText={displayedTextPhoneNumber(t('8005389552'))}
-          a11yLabel={a11yLabelID(t('8005389552'))}
-          numberOrUrlLink={t('8005389552')}
-          linkType={LinkTypeOptionsConstants.call}
-        />
+        <ClickToCallPhoneNumber phone={t('8005389552')} displayedText={displayedTextPhoneNumber(t('8005389552'))} a11yLabel={a11yLabelID(t('8005389552'))} />
```

Comment:
With our other uses of ClickToCallPhoneNumber, we allow it to generate the a11yLabel itself. In that case it puts spaces between the digits to ensure each one is announced as a single digit. Does a11yLabel need to be specified here?

Reply:
your right. this does the same thing. I removed the a11yLabel